agent-install/edgeNodeFiles.sh

```diff
@@ -139,6 +155,11 @@ function checkPrereqsAndInput () {
     if [[ -n $DIR && ! -d $DIR ]]; then
         fatal 1 "the value of option '-f' isn't an existing directory ..."
     fi
+    if [[ -n $PULL_REGISTRY ]]; then
+        if [[ -z $REGISTRY_USERNAME || -z $REGISTRY_PASSWORD ]]; then
+            fatal 1 "REGISTRY_USERNAME or REGISTRY_PASSWORD not set. When using the '-r' flag you must supply the username and password for the entitled registry you are attempting to pull the agent images from ..."
```

Comment:
Remove "entitled" from this message, because it open source that doesn't have much meaning.